src/System.Net.Primitives/tests/FunctionalTests/LoggingTest.cs

```diff
@@ -25,7 +25,10 @@ public void EventSource_ExistsWithCorrectId()
         }
 
         [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "NetEventSource is only part of .NET Core")]
+        [SkipOnTargetFramework(
+            TargetFrameworkMonikers.NetFramework | TargetFrameworkMonikers.UapAot | TargetFrameworkMonikers.Uap, 
```

Comment:
>Why are we not calling these UWP which is the official name instead of UAP?

In terms of NuGet monikers, UAP is the term such as 'uap10.1'

cc: @ericstj @weshaggard